tool/tctl/common/resource_command.go

```diff
@@ -1380,10 +1381,57 @@ func (rc *ResourceCommand) getCollection(ctx context.Context, client auth.Client
 			return nil, trace.Wrap(err)
 		}
 		return &installerCollection{installers: []types.Installer{inst}}, nil
+	case types.KindDatabaseService:
+		dbServices, err := listDatabaseServices(ctx, client, rc.ref.Name)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+
+		if len(dbServices) == 0 && rc.ref.Name != "" {
+			return nil, trace.NotFound("Database Service %q not found", rc.ref.Name)
+		}
+
+		return &databaseServiceCollection{databaseServices: dbServices}, nil
 	}
 	return nil, trace.BadParameter("getting %q is not supported", rc.ref.String())
 }
 
+func listDatabaseServices(ctx context.Context, client auth.ClientI, name string) ([]types.DatabaseService, error) {
```

Comment:
nit: can we implement this method in the `api/client`  like `(c *Client) GetDatabaseServices`  and reused it here ?

Reply:
Actually, we should be using ` apiclient.GetResourcesWithFilters` instead
It already iterates over all pages and return the entire list.